requirements/gen_conda_requirements.py

```diff
@@ -53,7 +53,7 @@ def compute_requirements(requirement_names=('core', ), options=None):
         conda_reqs_lines.append('# Python 3 conda configuration')
 
     for req_name in requirement_names:
-        fname = os.path.join(REQS_DIR, '{}.txt'.format(req_name))
+        fname = os.path.join(REQS_DIR, '{0}.txt'.format(req_name))
```

Comment:
Not really a necessary change, but allows py2.6 (which is what my crappy system had on it before I remembered to activate an environment).

Reply:
Python 2.6 is [no longer supported](https://github.com/SciTools/iris/blob/master/INSTALL#L68)

Reply:
😄  I agree entirely. There isn't a chance that Iris will work on py26, and I have no desire for that. I just ran the "gen_conda_requirements" script on my machine so that I could conda install the dependencies, and it barfed so I quickly fixed it.

I don't *need* this change in the slightest, but equally, it doesn't do any harm IMO. I'd personally sooner merge it than roll it back, but if a reviewer insisted, I'd do it (no debate/discussion, it isn't worth it).